packages/example/pages/_app.tsx

```diff
@@ -108,55 +115,76 @@ const chains = [
-  projectId,
-  chains,
-});
+// If any of the alchemy HTTPS URLs are undefined, wagmi will default
+// to using the first available RPC HTTPS URL from the chain.
+const transports = {
+  [mainnet.id]: http(process.env.NEXT_PUBLIC_RPC_MAINNET_HTTPS),
+  [polygon.id]: http(process.env.NEXT_PUBLIC_RPC_POLYGON_HTTPS),
+  [optimism.id]: http(process.env.NEXT_PUBLIC_RPC_OPTIMISM_HTTPS),
+  [arbitrum.id]: http(process.env.NEXT_PUBLIC_RPC_ARBITRUM_HTTPS),
+  [base.id]: http(process.env.NEXT_PUBLIC_RPC_BASE_HTTPS),
+  [zora.id]: http(process.env.NEXT_PUBLIC_RPC_ZORA_HTTPS),
+  [bsc.id]: http(process.env.NEXT_PUBLIC_RPC_BSC_HTTPS),
+  [zkSync.id]: http(process.env.NEXT_PUBLIC_RPC_ZKSYNC_HTTPS),
+  // Testnet networks
+  [goerli.id]: http(process.env.NEXT_PUBLIC_RPC_GOERLI_HTTPS),
+  [sepolia.id]: http(process.env.NEXT_PUBLIC_RPC_SEPOLIA_HTTPS),
+  [holesky.id]: http(process.env.NEXT_PUBLIC_RPC_HOLESKY_HTTPS),
+  [polygonMumbai.id]: http(process.env.NEXT_PUBLIC_RPC_POLYGON_MUMBAI_HTTPS),
+  [optimismSepolia.id]: http(
+    process.env.NEXT_PUBLIC_RPC_OPTIMISM_SEPOLIA_HTTPS,
+  ),
+  [arbitrumSepolia.id]: http(
+    process.env.NEXT_PUBLIC_RPC_ARBITRUM_SEPOLIA_HTTPS,
+  ),
+  [baseSepolia.id]: http(process.env.NEXT_PUBLIC_RPC_BASE_SEPOLIA_HTTPS),
+  [zoraSepolia.id]: http(process.env.NEXT_PUBLIC_RPC_ZORA_SEPOLIA_HTTPS),
+};
 
-const wagmiConfig = getDefaultConfig({
+const config = getDefaultConfig({
   appName: 'RainbowKit Demo',
   projectId,
   chains,
+  transports,
   wallets: [
     ...wallets,
     {
       groupName: 'Other',
       wallets: [
-        argentWallet({ projectId, chains }),
-        bifrostWallet({ projectId, chains }),
-        bitgetWallet({ projectId, chains }),
-        bitskiWallet({ chains }),
-        clvWallet({ projectId, chains }),
-        coin98Wallet({ projectId, chains }),
-        coreWallet({ projectId, chains }),
-        dawnWallet({ chains }),
-        desigWallet({ chains }),
-        enkryptWallet({ chains }),
-        foxWallet({ projectId, chains }),
-        frameWallet({ chains }),
-        frontierWallet({ projectId, chains }),
-        imTokenWallet({ projectId, chains }),
-        ledgerWallet({ projectId, chains }),
-        mewWallet({ chains }),
-        oktoWallet({ projectId, chains }),
-        okxWallet({ projectId, chains }),
-        omniWallet({ projectId, chains }),
-        oneKeyWallet({ chains }),
-        phantomWallet({ chains }),
-        rabbyWallet({ chains }),
-        safeheronWallet({ chains }),
-        safepalWallet({ projectId, chains }),
-        subWallet({ projectId, chains }),
-        tahoWallet({ chains }),
-        talismanWallet({ chains }),
-        tokenPocketWallet({ projectId, chains }),
-        tokenaryWallet({ chains }),
-        trustWallet({ projectId, chains }),
-        uniswapWallet({ projectId, chains }),
-        xdefiWallet({ chains }),
-        zealWallet({ chains }),
-        zerionWallet({ projectId, chains }),
+        argentWallet({ projectId }),
```

Comment:
We'll want this to look like `argentWallet` without passing calling the function or passing the projectId.

Reply:
The walletConnectors should just be `Wallet` instead of `() => Wallet types` (and devs could just pass a list that looks like wallets: [argentWallet, rainbowWallet] like chains).